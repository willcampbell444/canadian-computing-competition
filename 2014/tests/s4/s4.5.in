100
1000000
11532502 346211462 32256617 409573192 20690
181372373 46161111 381174588 250560093 29863
231221681 73428574 358607697 137853721 20157
349343599 877331739 982687368 990556074 95650
200494603 103929056 624417759 629671203 79599
361912920 648279518 859069903 996741330 12727
84619809 570938743 810501999 618307466 96423
726940774 146133976 840747371 659104029 16428
452658609 91658721 844985022 353890337 32142
42809013 84042940 628855786 412151608 19695
127956528 2818624 655220978 934218643 72878
741163567 80076066 856084996 123896546 61434
16422069 3360208 666792411 920785445 31351
317911195 194123823 601789251 416942766 16562
223602224 146117536 368889833 779175097 59067
228073448 765109220 511998608 986609706 65498
494344835 11945727 762797754 986878442 63845
157831663 327434574 224109093 708187792 60183
41202518 104741585 650826797 793964270 82011
179679707 309321065 217871658 514811061 96909
255527960 484108299 740109650 590582704 68273
53164584 146256260 595443986 183949330 4471
28089035 14184494 862039232 793475745 76924
98261322 63097505 257818201 321483140 11758
145983966 803075369 243770715 919170266 82723
469869363 403850670 846346317 589992500 27880
319034388 537271136 894713957 569957583 26793
32402857 719945225 196512570 749893483 11379
72874154 168717805 961717789 628935594 96403
134077412 725271569 178031178 728700975 63005
737718445 410794177 842210233 755647232 86313
219424058 423699485 854449522 509951959 22930
319836598 135112723 742299160 882455275 19322
326279535 254377147 469747031 888446839 8676
47217504 6819891 978546832 601614302 53325
118159792 282229248 783061753 932072433 55145
603678343 309145566 792863320 517772799 64248
187340886 220223078 234233471 424405340 26744
504821999 4306051 549122154 628426475 72573
438705298 8566528 860215311 778803567 10326
259362788 739949853 619232803 866041346 21435
111346145 17012774 507958858 729381056 29695
31466545 36221179 67883117 370223231 96637
146292469 16217507 423190943 897318716 73772
26048962 630387183 65644643 865707186 45256
124779025 655524020 222456902 857101066 78224
146710918 252150648 703683983 417236830 72934
99366368 200051513 918202211 455736450 24880
615734306 715764628 800179142 861220658 96805
287821186 70478635 709419265 380361905 87057
172705035 48611080 962618894 741421800 67638
653630715 192555793 710022363 572207141 35501
533734761 440917802 686899257 636704706 63935
28364577 568049968 280001529 642005904 96193
49437904 236092337 472659316 619659718 6882
39459315 206234245 933328036 223726557 56181
288129945 259406212 731469644 817340209 84868
484342347 378787092 926104114 863349121 40709
374709529 113979476 683613971 255477725 92301
20431865 388190098 425635523 948438375 40694
430412794 222888855 585833251 679906641 84196
282671431 19667097 446256892 624896847 91605
16809223 630743521 839206728 667862579 59049
87560751 400222440 412745254 430282746 51152
302222479 260547474 778490238 654362592 10651
477297142 869584312 854934830 874622627 6853
58817811 146962174 387047562 380390965 14830
73458495 57621100 763273157 961295280 69744
412881314 161333805 960305554 569559519 99733
48641842 125147566 968634543 914703134 22844
7176458 17713716 423373062 355628568 35969
222702219 39664851 365710504 398226208 16998
807292413 361886739 965127826 959368985 17263
160772456 418115481 268535805 568223260 74762
508225638 308807182 952056255 998812325 99916
184265058 64389410 654034361 534183174 55890
321770688 66158465 675216860 636113264 16476
132893196 89440351 339068130 738926255 15763
560277659 18289557 701753585 392296965 96942
78211681 243713797 626568375 622340577 54544
429826473 748380205 475245260 751521782 63535
145326636 254354930 450834173 640617573 21932
591778395 36652967 627879533 815694086 76521
164339785 394779647 484524739 942277528 83312
143992103 8671791 543165876 828873114 77305
342714340 55684473 841193583 241749704 46198
192935977 556481191 741581400 767637048 77646
443279384 575818639 880931622 858487114 35485
30345093 38006006 607052587 56336905 82877
444815318 403103632 790696670 620457613 78700
281421341 768712445 973689601 882466071 67694
502288999 212973030 729942501 362517575 59680
299936442 652044327 818889853 887644379 65338
152059208 153552120 300296757 501007643 3987
210359450 511991585 366820049 753176382 78760
89367136 687837493 171013743 695364077 51629
233357487 527123165 781968858 743655345 41069
194573644 450276580 355444565 729107167 91226
546173077 140085448 773930419 185781054 30618
604396550 288144118 824130040 677037709 45820