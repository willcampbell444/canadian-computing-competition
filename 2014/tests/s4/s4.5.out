341429836801334897
